core-domain/src/test/java/org/uniprot/core/uniref/impl/UniRefEntryBuilderTest.java

```diff
@@ -78,20 +83,13 @@ void testUpdated() {
         assertEquals(update, entry.getUpdated());
     }
 
-    @Test
-    void testCommonTaxonId() {
-        long commonTax = 9606L;
-
-        UniRefEntry entry = new UniRefEntryBuilder().commonTaxonId(commonTax).build();
-        assertEquals(commonTax, entry.getCommonTaxonId());
-    }
-
     @Test
     void testCommonTaxonName() {
-        String commonTax = "Homo sapiens";
+        Organism organism =
+                new OrganismBuilder().taxonId(9606L).scientificName("scientific").build();
 
-        UniRefEntry entry = new UniRefEntryBuilder().commonTaxon(commonTax).build();
-        assertEquals(commonTax, entry.getCommonTaxon());
+        UniRefEntry entry = new UniRefEntryBuilder().commonTaxon(organism).build();
+        assertEquals(organism, entry.getCommonTaxon());
     }
 
```

Comment:
Could we add test case without either taxonId or scientificName or both?

Reply:
This use case is covered in OrganismBuilderTest, right?

Reply:
Oh sorry, i didnt see that. If that case is covered please ignore the comment